src/test/mon/MonMap.cc

```diff
@@ -96,31 +98,31 @@ TEST_F(MonMapTest, build_initial_config_from_dns) {
 
 
 
-  CephContext *cct = (new CephContext(CEPH_ENTITY_TYPE_MON))->get();
+  boost::intrusive_ptr<CephContext> cct = new CephContext(CEPH_ENTITY_TYPE_MON);
```

Comment:
Drop this change from `test/mon: fix compiler errors in MonMap unittest ` and the others. This is causing the build failure.